pandas/tests/indexing/test_indexing.py

```diff
@@ -571,17 +570,23 @@ def test_astype_assignment_with_dups(self):
         # result = df.get_dtype_counts().sort_index()
         # expected = Series({'float64': 2, 'object': 1}).sort_index()
 
-    def test_coercion_with_contains(self):
+    @pytest.mark.parametrize("val,expected", [
+        (np.inf, False),
+        (np.nan, False),
+        (3.0, True),
+        (3.5, False),
+        (0.0, True),
+    ])
+    def test_coercion_with_contains(self, val, expected):
```

Comment:
Ah, I guess I didn't understand that because I was explicitly using a `pd.Int64Index`, which should and does raise a `TypeError` if you try to initialize it with `np.inf`. I guess it should be fine if I use a normal `pd.Index`.